### ISSUE TITLE
Android 11 300.d Lenovo K9 stuck on OEM logo

### ISSUE THREAD

**Issue by samkoladziansky** [phhusson/treble_experimentations#1505]
Hello I have Lenovo K9 (mtk6762) (A-Only) (Android 8.1) and after flash android 11 gsi I get stuck on the OEM logo but after few minutes display get darker and charge led turn on (of course when is connected on charger). Please help. some  logs from /sys/fs/pstore/... and from TWRP recovery option...
![Screenshot_2009-12-31-18-14-27](https://user-images.githubusercontent.com/66882669/94835907-9d4e2a00-0412-11eb-891e-3609335bdf38.png)
[recovery.log](https://github.com/phhusson/treble_experimentations/files/5313450/recovery.log)
[dmesg.log](https://github.com/phhusson/treble_experimentations/files/5313451/dmesg.log)
[pmsg-ramoops-0.log](https://github.com/phhusson/treble_experimentations/files/5313454/pmsg-ramoops-0.log)
[console-ramoops-2.log](https://github.com/phhusson/treble_experimentations/files/5313455/console-ramoops-2.log)
[console-ramoops.log](https://github.com/phhusson/treble_experimentations/files/5313456/console-ramoops.log)

**Comment by samkoladziansky** [comment]
edit: I have flashed magisk because verified boot

**Comment by thebestgamer2** [comment]
Hi colleague!
I also have a lenovo k9 and with the same problem. Can you give me a link to your TWRP Recovery, because mine does not respect internal storage (internal storage (0)) and I have to delete data in ext4 format. Thanks in advance!

**Comment by samkoladziansky** [comment]
Iam using this twrp from the xda forum https://forum.xda-developers.com/android/general/bliss-rom-gsi-android-9-pie-lenovo-k9-t4046595/amp/ and the recovery have one small problem internal storage description didn't work that's why you have internal storage 0mb.

**Comment by thebestgamer2** [comment]
Yes, I created this forum and TWRP. :D

**Comment by samkoladziansky** [comment]
I know :)

**Comment by samkoladziansky** [comment]
I tried made recovery and boot but there is problem Lenovo didn't upload any K9 or K5s (chinese version with ZUI) kernel source and sources on GitHub is compiling error

**Comment by samkoladziansky** [comment]
Because sources on GitHub is not original for K9

**Comment by thebestgamer2** [comment]
Yes, I'm now trying to flash android 11. It starts, but I only see the lock screen, then the screen darkens and the lock screen reappears

**Comment by thebestgamer2** [comment]
I also tried different patches to start, but to no avail

**Comment by thebestgamer2** [comment]
From 4pda

**Comment by samkoladziansky** [comment]
What version you flash? Because after version 300.d I didn't try anymore

**Comment by thebestgamer2** [comment]
v300.I

**Comment by thebestgamer2** [comment]
this is the next version

**Comment by samkoladziansky** [comment]
Okay I try but how can I find patches for this?

**Comment by thebestgamer2** [comment]
I searched, tried with different patches to start, but nothing.
I'm using android 10 with a patch to run from 4pda

**Comment by thebestgamer2** [comment]
I'm happy with android 10, but I want 11 and whenever a new treble gsi for arm64 a appears, I go and flash it, but for now only the phhusson boot, but I only see the locked screen, nothing more

**Comment by samkoladziansky** [comment]
Interesting because I use Android 10 without any patches and works perfectly, and I want android 11 because is android 11 :) but I go try to boot and try to modify to work somehow.

**Comment by thebestgamer2** [comment]
I understand

**Comment by thebestgamer2** [comment]
I also tried nitrogen os with android 11, but it sends me to fastboot

**Comment by samkoladziansky** [comment]
Interesting

**Comment by samkoladziansky** [comment]
The kernel didn't have support for disabling selinux by cmd line but a know how to disable on the boot, you must open /bin/phh-on-boot.sh and add line setenforce 0 and selinux disabled I go try on the android 11.

**Comment by thebestgamer2** [comment]
this is the link
https://sourceforge.net/projects/amber-gsi/files/Nitrogen-OS-11/Nitrogen-OS-11-arm64-a-vanilla.img.xz/download

**Comment by samkoladziansky** [comment]
Thanks

**Comment by thebestgamer2** [comment]
:)

**Comment by thebestgamer2** [comment]
but the rom doesn't boot. send me to fastboot

**Comment by samkoladziansky** [comment]
Are you use magical when are you trying aosp11 because I have modified vendor (by me) and when I uninstall magisk I have EIO

**Comment by samkoladziansky** [comment]
*magisk

**Comment by thebestgamer2** [comment]
No, I haven't tried with magisk
